styleguide/_themes/global/scss/modifiers.scss

```diff
@@ -53,6 +53,26 @@ img { max-width: 100%; }
   padding: $standard-padding-full;
 }
 
+//equal height columns
+.row-eq-height {
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display:         flex;
+}
+
+@media only screen and (max-width: $screen-sm) {
```

Comment:
This should probably be a `max-width: $screen-xs-max` instead to prevent one off errors